com.unity.render-pipelines.universal/Runtime/ForwardRenderer.cs

```diff
@@ -51,6 +53,10 @@ public sealed class ForwardRenderer : ScriptableRenderer
 
         public ForwardRenderer(ForwardRendererData data) : base(data)
         {
+#if ENABLE_VR && ENABLE_XR_MODULE
+            UniversalRenderPipeline.m_XRSystem.InitializeXRSystemData(data.xrSystemData);
```

Comment:
Suggestion: move this to ScriptableRenderer base class and profit from not duplicating this code across renderers.

Reply:
Yes, we want to do this in future PR. 
The prerequisite is that we need a way to hold common shaders in URP(shaders used by all renderers).
Current URP shaders are per renderer.

We log this into jira so that we could revisit.